Doc/reference/expressions.rst

```diff
@@ -1156,7 +1156,8 @@ a user-defined function:
    first thing the code block will do is bind the formal parameters to the
    arguments; this is described in section :ref:`function`.  When the code block
    executes a :keyword:`return` statement, this specifies the return value of the
-   function call.
+   function call.  If execution falls off the end of the code block without
+   executing a :keyword:`return` statement, the return value is ``None``.
```

Comment:
```suggestion
   executing a :keyword:`return` statement, the return value is :const:`None`.
```

Reply:
:shrug: I followed the way it is formatted in the preceding paragraph. As a data point, there are 1385 instances of \`\`None\`\` and 126 of :const:\`None\` in _Doc/**_.

Reply:
We try to use links *everywhere* possible, not just in one place or paragraph (sometimes we prefix the reference with `~` to prevent it from link-ifying for redundancy, but that's besides the point--it's a good idea to always use the reference).

Reply:
Ah, nevermind. We have a [special case](https://devguide.python.org/documentation/markup/#quick-reference) for constants.